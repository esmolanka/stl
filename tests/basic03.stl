
type Foo = Unit
type Bar = Foo

#check Bar <: Foo
#check Foo <: Bar
 
