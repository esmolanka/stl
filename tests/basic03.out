tests/basic03.stl:5:1-5:18:
  Subtype:
    Unit
  Supertype:
    Unit
  Result:
    OK
tests/basic03.stl:6:1-6:18:
  Subtype:
    Unit
  Supertype:
    Unit
  Result:
    OK